example/schema/example.js

```diff
@@ -118,19 +118,6 @@ export const exampleDataType = parseField(DATA, {
   }
 });
 
-/*
```

Comment:
Uh, you didn't see anything...

But these were parts of the example that I hadn't re-implemented yet and forgot to take care of.